### ISSUE TITLE
ubi8-asciidoctor failing to build

### ISSUE THREAD

**Issue by garethahealy** [redhat-cop/containers-quickstarts#555]
- https://github.com/redhat-cop/containers-quickstarts/runs/5198712110?check_suite_focus=true

```
ERROR:  Error installing asciidoctor-epub3:
	nokogiri requires Ruby version < 3.2.dev, >= 2.6. The current ruby version is 2.5.0.
error building at STEP "RUN dnf install -y     python3-devel     git     make     gcc     redhat-rpm-config     ruby-devel     zlib-devel     libjpeg-turbo-devel     java-11-openjdk-headless     fontconfig     && gem install --no-document     "asciidoctor:${ASCIIDOCTOR_VERSION}"     "asciidoctor-confluence:${ASCIIDOCTOR_CONFLUENCE_VERSION}"     "asciidoctor-diagram:${ASCIIDOCTOR_DIAGRAM_VERSION}"     "asciidoctor-epub3:${ASCIIDOCTOR_EPUB3_VERSION}"     asciimath     "asciidoctor-pdf:${ASCIIDOCTOR_PDF_VERSION}"     "asciidoctor-revealjs:${ASCIIDOCTOR_REVEALJS_VERSION}"     coderay     epubcheck-ruby:4.2.4.0     haml     "kramdown-asciidoc:${KRAMDOWN_ASCIIDOC_VERSION}"     rouge     slim     thread_safe     tilt     "asciidoctor-bibtex:${ASCIIDOCTOR_BIBTEX_VERSION}"     && rm -rf /usr/local/share/gems/cache     && dnf clean all     && rm -rf /var/lib/dnf": error while running runtime: exit status 1
time="2022-02-15T10:49:14Z" level=error msg="exit status 1"
```

CC @buuhsmead

**Comment by garethahealy** [comment]
similar to Jenkins ruby failing due to ruby version:
- https://github.com/redhat-cop/containers-quickstarts/pull/552

**Comment by pabrahamsson** [comment]
Closed by #556
